scripts/fixup_firestore_v1_keywords.py

```diff
@@ -55,6 +55,7 @@ class firestoreCallTransformer(cst.CSTTransformer):
     'run_query': ('parent', 'structured_query', 'transaction', 'new_transaction', 'read_time', ),
     'update_document': ('document', 'update_mask', 'mask', 'current_document', ),
     'write': ('database', 'stream_id', 'writes', 'stream_token', 'labels', ),
+
```

Comment:
Odd that `black` didn't correct this -- do we not run it against the `scripts` directory?